frontend/components/Filters/DoubleSelect/index.tsx

```diff
@@ -0,0 +1,177 @@
+import {
+  Fragment,
+  Key,
+  ReactChild,
+  ReactFragment,
+  ReactPortal,
+  useState,
+} from "react";
+import { Listbox, Transition } from "@headlessui/react";
+import { CheckIcon, SelectorIcon } from "@heroicons/react/solid";
+import { Option } from "types/filters";
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(" ");
+}
+
+export default function FilterDoubleSelect({
```

Comment:
Honestly this component should not really be about `Filtering`, please make it a generale purpose component `RangeSelect` maybe?